ext/telemetry/lib.rs

```diff
@@ -322,8 +343,69 @@ mod hyper_client {
   }
 }
 
-static OTEL_PROCESSORS: OnceCell<(SpanProcessor, LogProcessor)> =
-  OnceCell::new();
+enum MetricProcessorMessage {
+  ResourceMetrics(ResourceMetrics),
+  Flush(tokio::sync::oneshot::Sender<()>),
+}
+
+struct MetricProcessor {
+  tx: tokio::sync::mpsc::Sender<MetricProcessorMessage>,
+}
+
+impl MetricProcessor {
+  fn new(exporter: MetricExporter) -> Self {
+    let (tx, mut rx) = tokio::sync::mpsc::channel(2048);
```

Comment:
Is there any reasoning why buffer of this size was chosen?

Reply:
its the default from the other two processors